packaging/opensuse-42.2/snapd.spec

```diff
@@ -149,6 +149,15 @@ go install -s -v -p 4 -x -tags withtestkeys github.com/snapcore/snapd/cmd/snapd
 %gobuild cmd/snapctl
 %gobuild cmd/snap-update-ns
 
+# Link all the libraries needed for seccomp
+ln -s /usr/include/libseccomp/* /usr/include/
```

Comment:
I don't think we want to do this. What is probably missing is appropriate call to pkg-config for seccomp CFLAGS. On a suse machine you may want to see what kind of things pkg-config tells us about it. It is possible that the pkg-config file has a different name but I don't know from the top of my head and I'm away from my physical suse box to check.